src/search/structures.go

```diff
@@ -0,0 +1,35 @@
+package search
+
+const DefaultUserAgent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
+
+type Dom struct {
+	Result      string
+	Link        string
+	Title       string
+	Description string
+	NextPage    string
+}
+
```

Comment:
Dom should be renamed so its purpose is more clear. Maybe something like `DomQuery`